cmd/lk/sip.go

```diff
@@ -643,6 +654,9 @@ func updateSIPOutboundTrunk(ctx context.Context, cmd *cli.Command) error {
 		tr := livekit.SIPTransport(trv)
 		req.Transport = &tr
 	}
+	if val := cmd.String("destination-country"); val != "" {
+		req.DestinationCountry = &val
```

Comment:
Is this like the 2-digit country code? Is there some validation? Maybe, worth adding in help the format needed for this.

Reply:
Updated the documentation